### PR TITLE
MSTLinkNodes created for each word pair. Created new atom type.

### PR DESCRIPTION
**Pull request by rohit12**
There is a storage directory also in my directory. I have added it to the .gitignore file since I think there is no point in uploading storage to the repository.

### CODE REVIEW

File: lib/opencog.conf [hunk 1]
```diff
@@ -107,9 +107,9 @@ PYTHON_PRELOAD = pyshell, restapi, pattern_match_functions, agent_finder
 # For information on how to set up atomspace persistance, see the README
 # file in opencog/persist/sql/README.
 #
-# STORAGE               = "opencog-atomspace"
-# STORAGE_USERNAME      = "opencog_user"
-# STORAGE_PASSWD        = "asdf"
+STORAGE               = "OpenCogTut"
+STORAGE_USERNAME      = "rohits134"
+STORAGE_PASSWD        = "rohits134"
 #
```
Comment (williampma) [inline lib/opencog.conf]: This looks specific to your local computer's configuration, so perhaps it shouldn't be part of the pull request?

File: opencog/nlp/learn/misc-scripts/config-fr.scm [hunk 2]
```diff
@@ -6,10 +6,10 @@
...
 ; The scheme shell listen port.
-(define shell-port 18002)
+(define shell-port 17001)
 (define shell-prompt "[0;34mscheme-fr[1;34m> [0m")
 
 ;--------------------------------------------------------------
 ; The relex server host and port
-(define relex-server-host "10.1.1.4")
+(define relex-server-host "127.0.0.1")
 (define relex-server-port 4444)
```
Comment (linas) [inline opencog/nlp/learn/misc-scripts/config-fr.scm]: these changes should not be checked in ...

File: opencog/nlp/learn/mst-parser.scm [hunk 3]
```diff
@@ -30,7 +30,6 @@
 ;
 (use-modules (srfi srfi-1))
 (use-modules (srfi srfi-11))
-
 ; ---------------------------------------------------------------------
```
Comment (linas) [inline opencog/nlp/learn/mst-parser.scm]: whitespace changes should not be checked in..

File: opencog/nlp/types/atom_types.script [hunk 4]
```diff
@@ -135,3 +135,8 @@ LG_OR  <- OR_LINK     // Exclusive-OR, completely disjunctive.
 // LgConnectors, joined with LgOrLink, LgAndLink.
 // Ideally, these are fully disjoined before use.
 LG_WORD_CSET <- LIST_LINK
+
+//MST parser node.
+//This comes from the MST parse
+MST_LINK_NODE <- PREDICATE_NODE "MSTLinkNode"
+MST_DISJUNCT_NODE <- PREDICATE_NODE "MSTDisjunctNode"
```
Comment (linas) [inline opencog/nlp/types/atom_types.script]: Missing newline

File: opencog/nlp/learn/make-disjuncts.scm [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+(define (create-MST-nodes text)
+	(map create-evaluation-link (filter criteria? (parse text)) ))
+
+;This function is used by the higher-order-function filter to selectively apply
+;the create-evaluation-link function to only those word pairs which do not have 
+;a mutual information of -1000
+(define (criteria? wp)
+	(if (= -1000 (get-mutual-information wp))
+		#f
+		#t))
```
Comment (linas) [inline opencog/nlp/learn/make-disjuncts.scm]: missing newline
